README.md

```diff
@@ -1,2 +1,48 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Awsome-Books-ES6
-Create an application to add and remove books using JavaScript ES6
+
+In this project, we will build a basic website that allows users to add/remove books from a list using JavaScript ES6 and Modules 
+
+## Built With
+
+- Javascript, HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+- Git clone the repository to your local machine:
+```
+https://github.com/AlejandraKeber/Awsome-Books-ES6.git
+```
+- Open the folder with you code editor
+
+
+## Authors:
+
+👤 **Alejandra Keber Maggi:** 
+
+- GitHub: [@AlejandraKeber](https://github.com/AlejandraKeber)
+- Twitter: @KeberAlejandra
+- LinkedIn: [alejandra-keber](www.linkedin.com/in/alejandra-keber)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job adding the MIT.md file to your project as requested by the previous reviewer, how ever, it is not correctly linked. kindly fix the link.